keyboards/ramonimbao/squishyfrl/keymaps/via/keymap.c

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 Ramon Imbao
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
VIA keymaps require 4 layers.

Reply:
Is it okay if I have to adjust the EEPROM memory usage?

Reply:
That's fine.  There is a define to lower the number of layers used.  But basically, the number of layers set needs to match the allowed layers, or you get layers filled with junk from the uninitialized EEPROM.  

Specifically, the define that controls this is `DYNAMIC_KEYMAP_LAYER_COUNT`, and defaults to 4. 

Also, changing this means that all users are limited to the number of layers specified.